docs/node/workers.md

```diff
@@ -7,25 +7,80 @@ sidebar_label: Workers
 ## What is a Worker?
 
 A Worker is an object that connects to the Temporal Server and executes [Workflows](/docs/node/workflows) and [Activities](/docs/node/activities).
-Workers are run on user-controlled hosts.
-You can use the `@temporalio/worker` package's [`Worker`](https://nodejs.temporal.io/api/classes/worker.Worker) class to create and run as many Workers as your use case demands, across any number of hosts.
 
-Workers poll [Task Queues](/docs/node/task-queues) for Tasks, execute chunks of code in response to those Tasks, and then communicate the results back to the Temporal Server.
-
-As a developer, running Workers is a fairly simple procedure because the Node SDK handles all of the communication between the Worker and the Temporal Server behind the scenes.
+- Workers are run on user-controlled hosts and automatically discovers Workflows and Activities based on `workDir` or specified paths, to be bundled using Webpack v5.
+- You can use the `@temporalio/worker` package's [`Worker`](https://nodejs.temporal.io/api/classes/worker.Worker) class to create and run as many Workers as your use case demands, across any number of hosts.
+- Workers poll [Task Queues](/docs/node/task-queues) for Tasks, execute chunks of code in response to those Tasks, and then communicate the results back to the Temporal Server.
 
 ## How to start a Worker
 
-To start a Worker, you need to pass the following two required options to the `Worker.create()` function:
+First you create a Worker with `Worker.create()`, then call `worker.run()` on it.
+
+Below is an example of starting a Worker that polls the Task Queue named `tutorial`.
+
+<!--SNIPSTART nodejs-hello-worker {"enable_source_link": false}-->
+<!--SNIPEND-->
+
+### Required Worker Options
+
+There are two required options to the `Worker.create()` function:
 
 1. The `workDir`. The Node SDK will automatically register:
```

Comment:
This is incorrect, `workDir` is not required, maybe move it below or above the table where we document `activities`, `workflowsPath` and `nodeModulesPath`?

Reply:
ok got you. thanks